observation_portal/requestgroups/pattern_expansion.py

```diff
@@ -54,11 +64,18 @@ def expand_mosaic_pattern(expansion_details):
         cos_dec = max(cos_dec, 10e-4)
         configuration_copy['target']['ra'] += (offset[0] / 3600.0 / cos_dec)
         extra_params = configuration_copy.get('extra_params', {})
-        extra_params['obs_recipe'] = 'Mosaic'
         configuration_copy['extra_params'] = extra_params
         configurations.append(configuration_copy)
 
     request_dict['configurations'] = configurations
+    # Save some information about the mosaic pattern inside the request
+    save_mosaic_params = expansion_details.copy()
+    save_mosaic_params.pop('request', None)
+    pattern = save_mosaic_params.pop('pattern', settings.MOSAIC['custom_pattern_key'])
+    if 'extra_params' not in request_dict:
+        request_dict['extra_params'] = {}
+    request_dict['extra_params']['mosaic_pattern'] = pattern
+    request_dict['extra_params']['mosaic_pattern_params'] = save_mosaic_params
```

Comment:
I believe the mosaic pattern params will have both the `*_overlap_percent` and `*_spacing` values set since we expand the percent into a value within the serializer before it gets here. Not sure if we want to "save" both since the user only inputs one of them...

Reply:
Removing pattern params since we decided not to store them